packages/x/x-markdown-css.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "x-markdown-css",
+  "filename": "x-markdown-css.min.css",
```

```suggestion
  "filename": "bundle.min.css",
```
Looking at the latest release, the file appears to be named `bundle`, not `x-markdown-css`.